lb_content_resolver/subsonic.py

```diff
@@ -135,16 +135,45 @@ def update_recordings(self, recordings):
             Updates recording_id, subsonic_id, last_update
         """
 
-        recordings = [(r[0], r[1], datetime.datetime.now()) for r in recordings]
+        recording_index = { r[0]:r[1] for r in recordings }
 
         cursor = db.connection().cursor()
         with db.atomic() as transaction:
-            cursor.executemany(
-                """INSERT INTO recording_subsonic (recording_id, subsonic_id, last_updated)
-                                        VALUES (?, ?, ?)
-                     ON CONFLICT DO UPDATE SET recording_id = excluded.recording_id
-                                             , subsonic_id = excluded.subsonic_id
-                                             , last_updated = excluded.last_updated""", recordings)
+
+            placeholders = ",".join(("?", ) * len(recording_index.keys()))
```

Comment:
When calculating length of a dict, no need to convert to keys first.
```suggestion
            placeholders = ",".join(("?", ) * len(recording_index))
```

Reply:
Ah yes. 'twas a leftover from when I was debugging something.